samples/ballerina-by-example/examples/content-based-routing-service/content-based-routing-service.bal

```diff
@@ -0,0 +1,54 @@
+import ballerina.net.http;
+import ballerina.lang.jsons;
+import ballerina.lang.messages;
+import ballerina.doc;
+
+@doc:Description {value : "Service is invoke using BasePath value (/cbr)."}
+@http:BasePath {value:"/cbr"}
+service contentBasedRouting {
+ @doc:Description {value : "http:POST{} annotation declares the HTTP method."}
+ @http:POST{}
+ @http:Path {value:"/route"}
+ resource cbrResource (message m) {
+   //Create service endpoint using HTTP client-connector.
+   http:ClientConnector locationEP = create
+              http:ClientConnector("http://www.mocky.io");
+   //Declare a string within inverted commas
+   string sanFranString = "sanFrancisco";
+   //Native function to get JSON payload from message.
+   //Try more native functions
+   //Eg: For XML payloads, getXmlPayload() is used.
+   json jsonMsg = messages:getJsonPayload(m);
+   //getString() Native function retrieve the string value relevant to the key(name).
+   //Some other natives: getFloat(), getInt(), getBoolean(), getJson().
+   string nameString = jsons:getString(jsonMsg, "$.name");
```

Comment:
Use the newer JSON syntax
i.e: ` jsonMsg.name ` or  ` jsonMsg["name"] `
Again, no need to explain the json syntax here.  Only explains what you are trying to do.(e.g: "get the name of the blah blah, from the json request")